examples/src/main/scala/io/chrisdavenport/shellfish/Place.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024 Typelevel
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package io.chrisdavenport.shellfish
+
+import cats.effect.{IO, IOApp}
+
+import scodec.codecs.*
+import scodec.Codec
+
+import syntax.path.*
+import fs2.io.file.Path
+
+object Place extends IOApp.Simple {
+
+  case class Place(number: Int, name: String)
+
+  implicit val placeCodec: Codec[Place] = (int32 :: utf8).as[Place]
+
+  val path = Path("src/main/resources/place.data")
+
+  def run: IO[Unit] =
+    for {
+      exists <- path.exists()
+      _      <- if (exists) IO.unit else path.createFile
```

Comment:
```suggestion
      _      <- path.createFile.whenA(exists)
```
nitpick, also I'm not sure We should use this one, as most of our target is typelevel "beginners"

Reply:
I can add a coment saying that it's the same as the first one :D